CHANGES.markdown

```diff
@@ -1,3 +1,7 @@
+## Changes in 0.7.0
+ - Add functions and orphan instances introduced by changes to
+   `base-4.7.0.0` and `base-4.8.0.0`
```

Comment:
OK. I bumped it from 0.7.0 to 0.7.1, since I don't think there were any breaking changes.